build_manifest.yml

```diff
@@ -83,16 +83,6 @@ circuits-x86_64-linux-clang-assert:
 l1-contracts:
   buildDir: l1-contracts
 
-# We should move the noir source code to the top of the repo.
-# It's currently under yarn-project which looks very circular, but it isn't.
-# We're specific with our rebuild patterns as we depend on aztec.nr, but it doesn't have its own build.
-noir-contracts-build:
-  buildDir: yarn-project
-  projectDir: yarn-project/noir-contracts
-  rebuildPatterns:
-    - ^yarn-project/noir-contracts/
-    - ^yarn-project/aztec-nr/
-
```

Comment:
Given that we're now building noir contracts as part of the `yarn-project` step, and not in this job, shouldn't we include `*.nr` and `*.toml` to the rebuild-patterns of `yarn-project`?

Reply:
oh yeah, good shout. I had this in the previous PR and undid the change when restoring Nargo